### ISSUE TITLE
no-repetitive-locators false positives when multple arguments are used

### ISSUE THREAD

**Issue by ext** [alecxe/eslint-plugin-protractor#89]
The rule `protractor/no-repetitive-locators` does not consider multiple arguments when comparing locators. Even if the first argument is the same the second might alter the selector. This is an issue with both some builtin locators such as [cssContainingText](http://www.protractortest.org/#/api?view=ProtractorBy.prototype.cssContainingText) and when custom locators are used.

Given the following source:

```js
element(by.cssContainingText('.pet', 'Dog'));
element(by.cssContainingText('.pet', 'Cat'));
```

and this configuration:

```yaml
---
root: true
plugins:
    - protractor
rules:
  protractor/no-repetitive-locators: "error"
```

## Expected results

No errors

## Actual results

    2:9  error  Repetitive locator detected  protractor/no-repetitive-locators

I believe this is because of [get-locator.js#L21](https://github.com/alecxe/eslint-plugin-protractor/blob/master/lib/get-locator.js#L21) which only considers `node.arguments[0]` but have not made any deeper examination of the issue.

**Comment by alecxe** [comment]
This is a very good thing to spot. Thanks so much for the report and PR!
